.github/workflows/tests.yml

```diff
@@ -14,9 +14,7 @@ jobs:
       uses: actions/setup-python@v1
       with:
         python-version: ${{ matrix.python-version }}
-    - name: Create docker network
-      run: docker network create -d bridge --attachable cl_net_overlay
     - name: Build Image
       run: docker-compose -f docker-compose.dev.yml up --build -d
     - name: Run tests
```

Comment:
IM surprised its not needed for tests but great

Reply:
Yeah, it uses a simpler kind of network if you don't create one manually.